apps/evm/test/evm_test.exs

```diff
@@ -47,9 +45,19 @@ defmodule EvmTest do
 
   test "Ethereum Common Tests" do
     for {test_group_name, _test_group} <- @passing_tests_by_group do
-      for {_test_name, test} <- passing_tests(test_group_name) do
+      for {test_name, test} <- passing_tests(test_group_name) do
         {gas, sub_state, exec_env, _} = run_test(test)
-        assert_state(test, exec_env.account_interface)
+
+        context = %{
```

Comment:
I think this whole test needs to be refactored, just maybe not as part of this PR.